packages/@aws-cdk/aws-rds/test/integ.cluster-snapshot.ts

```diff
@@ -0,0 +1,115 @@
+import * as path from 'path';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as iam from '@aws-cdk/aws-iam';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { App, ArnFormat, CustomResource, RemovalPolicy, Stack, StackProps } from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+import { Construct } from 'constructs';
+import * as rds from '../lib';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core';
+
+class TestStack extends Stack {
```

Comment:
@peterwoodworth please check the integ test here.

I'm doing the following:
* Create a new cluster with a generated secret
* Take a snapshot (using a custom resource)
* Create a cluster from this snapshot with `SnapshotCredentials.fromGeneratedSecret()` to change the master user password in this cluster
* Add rotation to this cluster

This results in a cluster created from the snapshot with new credentials. If I go in the Secrets Manager console I can click "Rotate secret immediately" and change the master user password of this cluster without any issues.

Reply:
@rix0rrr FYI